spec/tictactoe_spec.rb

```diff
@@ -1,11 +1,46 @@
-require 'spec_helper'
-require 'tictactoe'
+require "spec_helper"
+require "tictactoe"
+require "presenter"
+require "fake_standard_out"
 
 RSpec.describe "TicTacToe" do
+  class FakePresenter
+    def display_board
+      <<~BOARD
+          |  |
+        --+--+--
+          |  |
+        --+--+--
+          |  |
+      BOARD
+    end
```

Comment:
I wonder if we need the `FakePresenter` to display the same board. What benefit do we get from using a fake here vs the real thing? Is there some way we could change the `FakePresenter` to make its existence worthwhile?

Reply:
removed fakepresenter